.github/workflows/pr.yml

```diff
@@ -2,33 +2,98 @@ name: pr
 on: [pull_request]
 jobs:
   test:
-    name: 'Node ${{ matrix.node }}, React ${{ matrix.react }}'
+    name: 'Test'
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: [16.14.2]
-        react: [17, 18]
     steps:
       - uses: actions/checkout@v2
       - uses: pnpm/action-setup@v2.2.2
         with:
           version: 7
       - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: 16.14.2
           cache: 'pnpm'
       - name: Install dependencies
         run: pnpm --filter "./packages/**" --filter query --prefer-offline install
-      - run: |
-          pnpm run test:ci
-          pnpm run test:size
-        if: matrix.react == '18'
-        env:
-          REACTJS_VERSION: ${{ matrix.react }}
-          BUNDLEWATCH_GITHUB_TOKEN: ${{ secrets.BUNDLEWATCH_GITHUB_TOKEN }}
       - run: pnpm run test:jest
-        if: matrix.react == '17'
-        env:
-          REACTJS_VERSION: ${{ matrix.react }}
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
+  lint:
+    name: 'Lint'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
This can use v3 now, right?

Reply:
probably, yes. what's the difference? are there breaking changes?

Reply:
Node 16 by default and a minimum actions runner version: https://github.com/actions/checkout/releases/tag/v3.0.0

Reply:
done: [42f3c3d](https://github.com/TanStack/query/pull/4288/commits/42f3c3db718340f86f58a45a0f83523b6b7e031a)